2-spending-db/task.md

```diff
@@ -73,6 +94,9 @@ To submit this homework write the correct commands after each question.
 Can you work out how to do this?
 
 ```sql
-
+SELECT date as "Month", SUM(amount) AS "Monthly Spend"
+FROM spends
+WHERE date IN ('2021-03-01', '2021-04-01')
+GROUP BY date;

```

Comment:
this query treats date as full date: `yyyy-mm-dd`
but `SELECT date as "Month"` implies you only select Month + Year parts from it
the right solution would be: `group by` and `select` that expression which only has this part of the date: `yyyy-mm`. Here is the SQL documentation you need for that: [postgresql: Date/Time Functions and Operators](https://www.postgresql.org/docs/current/functions-datetime.html)